integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/e2e/AbstractE2ETest.java

```diff
@@ -22,4 +28,19 @@ protected void minimalSetupForClusterCreation(TestContext testContext) {
         createDefaultEnvironment(testContext);
         initializeDefaultClusterDefinitions(testContext);
     }
+
+    /**
+     * Checks the cloud platform when a specific cloud provider is the only one which is needed.
+     * It throws {@link AssertionError} if the cloud provider doesn't match.
+     *
+     * @param cloudPlatform the required/expected cloud platform. Must not be null.
+     */
+    protected void checkCloudPlatform(@Nonnull CloudPlatform cloudPlatform) {
```

Comment:
This is not working for AWS right now, we still need to figure out why.

Reply:
then I'll keep this check method just in case but remove the usage of it

Reply:
alright